contracts/src/CrossChainApplications/NativeTokenBridge/README.md

```diff
@@ -0,0 +1,34 @@
+# Native-to-Native Token Bridge
+
+A pair of smart contracts built on top of Teleporter to support using an ERC20 token or the native token of any `subnet-evm` chain as the native token for a given subnet.
+
+## Design
+The native-to-native bridge is implemented using two primary contracts.
+### `NativeTokenSource`
+    - Lives on the Source chain. Pairs with exactly one `NativeTokenDestination` contract on a different chain.
+    - Locks and unlocks native tokens on the Source chain corresponding to mints and burns on the destination chain.
+    - `transferToDestination`: transfers all tokens paid to this function call to `recipient` on the destination chain by locking them and instructing the destination chain to mint. Optionally takes the address of an ERC20 contract `feeContractAddress` as well as an amount `feeAmount` that will be used as the relayer-incentivisation for the teleporter cross-chain call. Also allows for the caller to specify `allowedRelayerAddresses`.
```

Comment:
```suggestion
    - `transferToDestination`: transfers all tokens paid to this function call to `recipient` on the destination chain by locking them and instructing the destination chain to mint. Optionally takes the address of an ERC20 contract `feeContractAddress` as well as an amount `feeAmount` that will be used as the relayer-incentivization for the teleporter cross-chain call. Also allows for the caller to specify `allowedRelayerAddresses`.
```

I know either could be correct, but thus far we've been consistent with our "ize" spellings throughout the code base :) There are also a handful of other similar spellings to update